blocks/minesweeper/src/types/generated/block-entity.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * This file was automatically generated – do not edit it.
+ */
+
+import { Entity } from "@blockprotocol/graph";
+
+export type BlockEntity = MinesweeperBlock;
+
+export type BlockEntityOutgoingLinkAndTarget =
+  MinesweeperBlockOutgoingLinkAndTarget;
+
+export type MinesweeperBlock = Entity<MinesweeperBlockProperties>;
+
+export type MinesweeperBlockOutgoingLinkAndTarget = never;
+
+export type MinesweeperBlockOutgoingLinksByLinkEntityTypeId = {};
+
+/**
+ * The block entity of the "Minesweeper" block
```

Comment:
```suggestion
 * The block entity of the "Minesweeper" block
 * 
 * See: https://blockprotocol.org/@hash/blocks/minesweeper
```

I know we can't edit this file but we should change the description to match this, it's consistent with the other block entity types

Reply:
https://github.com/hashintel/hash/pull/2222/commits/afca30260994fac67c35cca5eb0fd41f53dc9855